aim_spacy/aim_displacy.py

```diff
@@ -17,8 +17,10 @@ def __call__(self, docs, style, caption='', quality=90, optimize=False):
         image_list = []
 
         self.options.update(dict(jupyter=False, style=style, page=False))
+        manual = self.options.pop('manual', None)
```

Comment:
although this kinda muddies the options param to be different from spacy maybe should just add `manual` as kwarg to `__init__`